proposals/target-typed-new.md

```diff
@@ -46,54 +51,51 @@ The type of the expression would be inferred from the target-type which would be
 with the following exceptions:
 
 - **Enum types:** not all enum types contain the constant zero, so it should be desirable to use the explicit enum member.
-- **Delegate types:** if the type is inferrable, an anonymous function can already be used.
 - **Interface types:** this is a niche feature and it should be preferable to explicitly mention the type.
-- **Tuple types:** tuple literals should be used for such expressions.
 - **Array types:** arrays need a special syntax to provide the length.
+- **Struct default constructor**: this rules out all primitive types and most value types. If you wanted to use the default value of such types you could write `default` instead.
```

Comment:
The downside of this is that you can't use object initializer with default ctor, `(S)new() { x = 1 }`. `default` doesn't help here. Would that be enough reason to relax this? (note: now that we went with the late-filter, this restriction wouldn't make much difference in overload resolution).

Reply:
That's a good point. Let me raise the question of default constructor again.
I think the rationale for disallowing it was that you could just use `default`, but as your example shows, that's not the case (target-typed new would allow an initializer).

Reply:
Interestingly, I couldn't find notes for disallowing binding `new()` to default constructor. I'll ping Mads/Neal to confirm.

Reply:
Mads doesn't recall disallowing. Let's proceed under the assumption that it's allowed.